src/qibocal/protocols/resonator_utils.py

```diff
@@ -0,0 +1,360 @@
+import numpy as np
+from numpy.typing import NDArray
+from scipy.ndimage import gaussian_filter1d
+from scipy.optimize import leastsq, newton
+
+
+def get_cable_delay(frequencies: NDArray, z: NDArray, num_points: int = 10) -> float:
```

Comment:
```suggestion
def cable_delay(frequencies: NDArray, z: NDArray, num_points: int = 10) -> float:
```
in general, `get_` prefixes are not adding anything to the names, so you could even save characters.

Reply:
Done!